packages/block-library/src/paragraph/deprecated.js

```diff
@@ -325,25 +327,21 @@ const deprecated = [
 			);
 		},
 		migrate( attributes ) {
-			return migrateCustomColorsAndFontSizes(
-				omit( {
-					...attributes,
-					customFontSize: Number.isFinite( attributes.fontSize )
-						? attributes.fontSize
+			return migrateCustomColorsAndFontSizes( {
+				...attributes,
+				customFontSize: Number.isFinite( attributes.fontSize )
+					? attributes.fontSize
+					: undefined,
+				customTextColor:
+					attributes.textColor && '#' === attributes.textColor[ 0 ]
+						? attributes.textColor
 						: undefined,
-					customTextColor:
-						attributes.textColor &&
-						'#' === attributes.textColor[ 0 ]
-							? attributes.textColor
-							: undefined,
-					customBackgroundColor:
-						attributes.backgroundColor &&
-						'#' === attributes.backgroundColor[ 0 ]
-							? attributes.backgroundColor
-							: undefined,
-				} ),
-				[ 'fontSize', 'textColor', 'backgroundColor', 'style' ]
```

Comment:
I may be missing something, but it looks like your code isn't omitting these anymore?

Reply:
Thanks for double-checking - this is intentional. If you look at the original code, you'll notice that `omit()` is called with a single argument by mistake. The second argument was being passed to `migrateCustomColorsAndFontSizes()` and it just ignored it. So we never actually omitted them here and I preferred to keep the same behavior.

Reply:
Oh, you're right, the array is outside of the `omit` call! Missed that, sorry 👍